src/textual/css/styles.py

```diff
@@ -182,6 +183,9 @@ class StylesBase(ABC):
     layout = LayoutProperty()
 
     color = ColorProperty(Color(255, 255, 255))
+    # N.B. This one is a pseudo-property, in the sense that we cannot set it via CSS via a dedicated property:
```

Comment:
We can also set it via code can't we? `self.styles.color_auto = True` ?

Reply:
We can yes, what my comment was meaning is that we cannot set it via a CSS string or file - as there is no `process_color_auto` method in the StylesBuilder. 
That's why to me it's a pseudo-property, as it can only be set via another CSS property, "color" :slightly_smiling_face:

Reply:
Gotcha